src/restruct_poc/model/model.hpp

```diff
@@ -0,0 +1,59 @@
+#pragma once
+
+#include "node.hpp"
+
+#include "src/restruct_poc/solver/constraint.hpp"
+#include "src/restruct_poc/types.hpp"
+
+namespace openturbine {
+
+static Node GroundNode(-1, {0., 0., 0., 1., 0., 0., 0.});
```

Comment:
Adding a comment to describe the intent of a `GroundNode` might be helpful here. Also, -1 for the node number makes me a little uncomfortable -- why the special treatment?

Reply:
This should be renamed to `InvalidNode`. It is used for the unused node in `FixedBC` and `PrescribedBC`. The `-1` gets used in a bunch of the sparse index functions to know if the base node is unused.